drivers/interrupt_controller/intc_miwu.c

```diff
@@ -333,8 +328,8 @@ int npcx_miwu_manage_dev_callback(struct miwu_dev_callback *cb, bool set)
 									       \
 		/* Clear all MIWUs' pending and enable bits of MIWU device */  \
 		for (i = 0; i < NPCX_MIWU_GROUP_COUNT; i++) {                  \
-			NPCX_WKPCL(base, i) = 0xFF;                            \
 			NPCX_WKEN(base, i) = 0;                                \
+			NPCX_WKPCL(base, i) = 0xFF;                            \
```

Comment:
Again, I'm not familiar with Nuvoton Platform, however most SoCs will trigger interrupt immediately when it is enabled if the status register was dirty. That's why it's common to clear interrupt status register shortly before it's enabled. Changing the sequence here feels wrong.

Reply:
From the Nuvoton datasheet, bits set in the WKEN register enable the interrupt request, bits clear disable the interrupt request.  So this code is disabling all interrupts, then clearing any stale interrupts.  I believe the sequencing here is correct.

Reply:
As @keith-zephyr mentioned, the sequenced is disabling all interrupts, then clearing any stale interrupts for sure. If there is a level type interrupt issued, the original source cannot clear the pending bit successfully since it was trigger immediately.